packages/http-client-csharp/generator/Microsoft.TypeSpec.Generator.ClientModel/src/Providers/ScmModelProvider.cs

```diff
@@ -253,6 +261,25 @@ private void UpdateFullConstructorParameters()
                 }
             }
 
+            // Update the initializer to include the json patch parameter as an argument
+            if (FullConstructor.Signature.Initializer != null)
+            {
+                var updatedArguments = new List<ValueExpression>(FullConstructor.Signature.Initializer.Arguments.Count);
+                foreach (var argument in FullConstructor.Signature.Initializer.Arguments)
+                {
+                    if (argument is VariableExpression { Declaration.RequestedName: AdditionalBinaryDataParameterName })
```

Comment:
Is the argument at all connected to the underlying RawDataField? If so, could we use that instead?